### ISSUE TITLE
DataGridView's scrollbars shouldn't include their control type in their accessible names

### ISSUE THREAD

**Issue by Tanya-Solyanik** [dotnet/winforms#2965]
**Steps to reproduce:**

1. Create a new WinForms .NET 4.8 app in VS.
2. Add a DataGridView which shows a horizontal scroll bar when run.
3. Run the app and have the Accessibility Insights for Windows tool examine the UI.

**Result**: The accessible name of the scrollbar is "Horizontal Scroll Bar", and so given that this includes the same (case insensitive) text as its localized control type of "scroll bar", Accessibility Insights reports this as an error.

**Expected**: A control shouldn't include its control type in its accessible name, and so the accessible name should be "Horizontal". (It's a similar case for the vertical scroll bar too.)

from @GuyBark: I spent a while trying to figure out how the accessible name is ending up as "Horizontal Scroll Bar". In https://referencesource.microsoft.com/#System.Windows.Forms/winforms/Managed/System/WinForms/DataGridView.cs, it sets this.horizScrollBar.AccessibleName to SR.GetString(SR.DataGridView_AccHorizontalScrollBarAccName), and in the code I found, that's "Horizontal" as it should be. But it seems that by the time a UIA client gets supplied the name, its "Horizontal Scroll Bar".

ported from AzDo 1071939

**Comment by Ashley-Li** [comment]
Verified the issue with latest NET SDK 6.0.100-alpha.1.20557.2, the issue has been fixed that get correct information("Vertical" scroll bar) about the DataGridView using inspect app.
![image](https://user-images.githubusercontent.com/56664234/98623356-5af4f400-22c0-11eb-9422-8810aba9cf41.png)
